software/mathesar.yml

```
@@ -1,6 +1,6 @@
 name: Mathesar
 website_url: https://mathesar.org/
-description: An intuitive UI for managing data collaboratively, for users of all technical skill levels. Built on Postgres – connect an existing DB or set up a new one.
+description: Manage data collaboratively, for users of all technical skill levels. Built on Postgres – connect an existing DB or set up a new one.
```

```suggestion
description: Intuitive UI to manage data collaboratively, for users of all technical skill levels. Built on Postgres – connect an existing DB or set up a new one.
```

I think it makes sense to include this in the description